packages/trip-form/src/TripOptions/styled.ts

```diff
@@ -0,0 +1,259 @@
+import { CheckCircle } from "@styled-icons/boxicons-solid/CheckCircle";
+import { PlusCircle } from "@styled-icons/boxicons-regular/PlusCircle";
+import ScrollContainer from "react-indiana-drag-scroll";
+import styled, { css } from "styled-components";
+// eslint-disable-next-line prettier/prettier
+import type { ButtonProps } from "./Checkbox"
+
+export const TripOptionsContainer = styled.div`
+  background-color: #0d5eac;
+  color: white;
+  font-weight: 40;
+  max-width: 992px; /* Maximum mobile width */
+  min-height: 400px;
+`;
+export const TripOptionsSubContainer = styled.div`
+  max-width: 700px;
+  padding: 12px;
+`;
+
+export const TransitOptionsContainer = styled.div`
+  display: flex;
+  gap: 20px;
+  > button {
+    flex: 1;
+  }
+`;
+
+export const buttonIconCss = css`
+  height: 3em;
+  margin-bottom: 10px;
+  width: 3em;
+`;
+
+export const GreenCheck = styled(CheckCircle)`
+  ${buttonIconCss}
+  background-color: white;
+  border-radius: 50%;
+  clip-path: circle(40% at 50% 50%); /* Hides white stroke */
+  color: rgb(84, 174, 88);
+`;
+
+export const UncheckedIcon = styled(PlusCircle)`
+  ${buttonIconCss}
+`;
+
+export const Image = styled.img`
+  max-width: 100%;
+`;
+
+export const FeaturedOptionImageWrapper = styled.div`
+  padding: 1em;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
Sort CSS (also in other classes below).

Reply:
Sorted in 775500dafb258ffc2fc5d431b8619ebe3f21b54f